blackbox/test_graph_generation.sh

```diff
@@ -6,6 +6,7 @@ TMP=$(mktemp -d -t converge.graphviz.XXXXXXXXXX)
 
 "$ROOT"/converge server --no-token &
 PID=$!
+sleep 0.25
```

Comment:
Could you change this to 1 second?  Testing in the CI container locally seems to indicate than anything less that a second doesn't reliably fix the problem.